dashboard/app/helpers/levels_helper.rb

```diff
@@ -487,6 +488,31 @@ def azure_speech_service_options
       response = http.request(request)
       speech_service_options[:azureSpeechServiceToken] = response.body
       speech_service_options[:azureSpeechServiceRegion] = CDO.azure_speech_service_region
+      uri = URI.parse("https://westus.tts.speech.microsoft.com/cognitiveservices/voices/list")
```

Comment:
i mentioned this in [this comment](https://github.com/code-dot-org/code-dot-org/pull/34919#discussion_r434926380), but we should update this URI as well